build/build.lib.js

```diff
@@ -39,5 +39,9 @@ const entries = [
 ];
 
 entries.forEach(async entry => {
-    await build(profileLibConfig({ entry, mode: process.env.NODE_ENV || 'development' }));
+    if (process.env.NODE_ENV === 'production') {
+        await build(getLibConfig({ entry, minify: true }));
+    }
+
+    await build(getLibConfig({ entry, minify: false }));
```

Comment:
This won't work, because in order to both build the minified and the non-minified versions, we need to run the script twice cause vite/rollup only builds one version at the times, which is different from how webpack works.

Reply:
I think a solution would be to change the commands so it builds in both prod and non-prod mode:
```
    "prod": "cross-env NODE_ENV=development npm run build:all && cross-env NODE_ENV=production npm run build:all",
```

What do you think? This way it would build both versions of the bundle.

Reply:
Yeah makes sense, I am planning on improving the build scripts after this PR. This is what I have on a local branch:

<details>
<summary>View image</summary>

![image](https://user-images.githubusercontent.com/5924865/209622699-9f5c86c6-c802-4450-a6ad-5f0352637309.png)

</details>

Reply:
Ok, so I tried to implement something similar like this. Please take a look. Because vite works a bit differently I've had to organize the commands a bit differently, but now at least there's consistency.

Also:
* Fixed some styling issues
* Added autoprefixer
* Excluded vue from being bundled

Please let me know what you think!